packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Banner/banner.js

```diff
@@ -0,0 +1,309 @@
+import React, { useState } from 'react';
+import defaultClasses from './banner.css';
+import { mergeClasses } from '../../../../../classify';
+import { arrayOf, bool, oneOf, shape, string } from 'prop-types';
+import Button from '../../../../Button/button';
+import resolveLinkProps from '../../resolveLinkProps';
+import { Link } from '@magento/venia-drivers';
+
+const toHTML = str => ({ __html: str });
+
+/**
+ * Page Builder Banner component.
+ *
+ * This component is part of the Page Builder / PWA integration. It can be consumed without Page Builder.
+ *
+ * @typedef Banner
+ * @kind functional component
+ *
+ * @param {props} props React component props
+ *
+ * @returns {React.Element} A React component that displays a Banner.
+ */
+const Banner = props => {
+    const classes = mergeClasses(defaultClasses, props.classes);
+    const [hovered, setHovered] = useState(false);
+    const toggleHover = () => setHovered(!hovered);
+    const {
+        appearance = 'poster',
+        minHeight,
+        backgroundColor,
+        desktopImage,
+        mobileImage,
+        backgroundSize,
+        backgroundPosition,
+        backgroundAttachment,
+        backgroundRepeat,
+        textAlign,
+        border,
+        borderColor,
+        borderWidth,
+        borderRadius,
+        content,
+        showButton,
+        buttonType,
+        buttonText,
+        link,
+        linkType,
+        openInNewTab = false,
+        showOverlay,
+        overlayColor,
+        marginTop,
+        marginRight,
+        marginBottom,
+        marginLeft,
+        paddingTop,
+        paddingRight,
+        paddingBottom,
+        paddingLeft,
+        cssClasses = []
+    } = props;
+
+    let image = desktopImage;
+    if (
+        mobileImage &&
+        typeof window.matchMedia === 'function' &&
```

Comment:
Is it too soon to drop this check?  IE11 supports matchMedia and this is in a PWA.

Reply:
I did this originally as this API doesn't exist in JSDOM, but we can mock it so I'll remove it.